src/coverlet.console/Program.cs

```diff
@@ -223,7 +225,12 @@ static int Main(string[] args)
                         exceptionMessageBuilder.AppendLine($"The {dThresholdStat.ToString().ToLower()} method coverage is below the specified {dThreshold}");
                     }
 
-                    throw new Exception(exceptionMessageBuilder.ToString());
+                    if (dThresholdAct == ThresholdAction.Fail)
+                    {
+                        throw new Exception(exceptionMessageBuilder.ToString());
+                    }
+
+                    logger.LogWarning(exceptionMessageBuilder.ToString());
                 }
 
                 return exitCode;
```

Comment:
Are we sure that build doesn't fail in case of warning...returns a value different than 0, isn't this an error return value for a process?

Reply:
Nice catch. I made it so that it only changes the exit code when it should fail